js/src/Components/Atoms/Loader.js

```diff
@@ -3,7 +3,7 @@ import CircularProgress from '@material-ui/core/CircularProgress';
 
 export const Loader = () => {
   return (
-    <div>
+    <div style={{position: 'absolute', left: '50%', top: '50%'}}>
```

Comment:
To actually center shouldn't this be `50% - (width of div in px/2)`

Reply:
yeah i figured it was close enough.